nixos/modules/services/monitoring/longview.nix

```diff
@@ -35,10 +23,27 @@ in
 
       apiKey = mkOption {
         type = types.str;
+        default = "";
         example = "01234567-89AB-CDEF-0123456789ABCDEF";
         description = ''
           Longview API key. To get this, look in Longview settings which
           are found at https://manager.linode.com/longview/.
+
+          Warning: this secret is stored in the world-readable Nix store!
+          Use <option>apiKeyFile</option> instead.
+        '';
+      };
+
+      apiKeyFile = mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        example = "/run/keys/longview-api-key";
```

Comment:
We use /run/keys in all the examples of  https://github.com/NixOS/nixpkgs/issues/24288 because that's what nixops is using. See: https://nixos.org/nixops/manual/#idm140737318306400